test/heisenberg.jl

```diff
@@ -24,14 +24,13 @@ end
 H = square_lattice_heisenberg()
 χbond = 2
 χenv = 16
-verbosity = 1
-ctm_alg = CTMRG(; trscheme=truncdim(χenv), tol=1e-10, miniter=4, maxiter=100, verbosity)
+ctm_alg = CTMRG(; trscheme=truncdim(χenv), tol=1e-10, miniter=4, maxiter=100, verbosity=1)
 opt_alg = PEPSOptimize(;
     boundary_alg=ctm_alg,
-    optimizer=LBFGS(4; maxiter=100, gradtol=1e-3, verbosity),
+    optimizer=LBFGS(4; maxiter=100, gradtol=1e-3, verbosity=2),
     gradient_alg=GMRES(; tol=1e-6, maxiter=100),
     reuse_env=true,
-    verbosity,
+    verbosity=2,
```

Comment:
these changes can be reverted I think?

Reply:
mostly did this to have the CTM on verbosity 1 and LBFGS on 2 which gives the most informative output imo, dont know if you want this in a test tho